tensorflow_datasets/core/download/kaggle.py

```diff
@@ -93,9 +95,14 @@ def __init__(self, competition_name):
   @utils.memoized_property
   def competition_files(self):
     """List of competition files."""
+    if '/' in self._competition_name:
+      type = 'datasets'
+    else:
+      type = 'competitions'
```

Comment:
I'm not sure I understand. Where is `type` used ?

(Additionally `type` is a python built-in that your are redefining here.)